providers/apis/defi/uniswapv3/utils.go

```diff
@@ -84,15 +88,30 @@ func (pc PoolConfig) MustToJSON() string {
 	return string(b)
 }
 
-// DefaultETHAPIConfig is the default configuration for the Uniswap API. Specifically this is for
-// Ethereum mainnet.
-var DefaultETHAPIConfig = config.APIConfig{
-	Name:             fmt.Sprintf("%s%s%s", BaseName, NameSeparator, constants.ETHEREUM),
-	Atomic:           true,
-	Enabled:          true,
-	Timeout:          1000 * time.Millisecond,
-	Interval:         2000 * time.Millisecond,
-	ReconnectTimeout: 2000 * time.Millisecond,
-	MaxQueries:       1,
-	Endpoints:        []config.Endpoint{{URL: URL}},
-}
+var (
+	// DefaultETHAPIConfig is the default configuration for the Uniswap API. Specifically this is for
+	// Ethereum mainnet.
+	DefaultETHAPIConfig = config.APIConfig{
```

Comment:
nit: could change provider default name from `BaseName` to smth that doesn't contain `Base`